README.md

```diff
@@ -24,13 +24,13 @@ Currently we provide support for many platforms of the high level wallet library
 
 ### Android
 
-| Target                               | released binaries |
-|--------------------------------------|:------:|
-| `aarch64-linux-android`              |   ✓    |
-| `arm-linux-androideabi`              |   ✓    |
-| `armv7-linux-androideabi`            |   ✓    |
-| `i686-linux-android`                 |   ✓    |
-| `x86_64-linux-android`               |   ✓    |
+| Target                    | released binaries |
+| ------------------------- | :---------------: |
```

Comment:
is this auto-formatting? not really important, but I think it would be good to agree in a formatter/formatting rules to not have diffs all the time